src/app/model-download/components/model-download-component/model-download-component.component.ts

```diff
@@ -171,9 +169,6 @@ export class ModelDownloadComponentComponent implements OnInit {
     return findModel !== undefined ? true : false;
   }
 
-  goBack() {
-    this.router.navigate(["../" + this.lastPage], { relativeTo: this.activatedRoute });
-  }
```

Comment:
why was this removed?

Reply:
we have another method for going back, not sure which approach is better
`  clickBack() {
    RouteManager.moveBack();
  }`

Reply:
Leaving the current clickBack() method and resoling the conversation